src/app/components/ATIAnalytics/canonical/getNoScriptTrackingPixelUrl/index.ts

```diff
@@ -0,0 +1,10 @@
+import { reverbUrlHelper } from '@bbc/reverb-url-helper';
+import { ATIAnalyticsProps } from '../../types';
+
+type ATIAnalyticsPropsExport = Pick<ATIAnalyticsProps, 'reverbParams'>;
+
+const getNoScriptTrackingPixelUrl = (
+  reverbParams: ATIAnalyticsPropsExport['reverbParams'],
+) => reverbUrlHelper.getTrackingPixelSrc(reverbParams);
+
+export default getNoScriptTrackingPixelUrl;
```

Comment:
Let's use the default export
```suggestion
export default (
  reverbParams: ATIAnalyticsPropsExport['reverbParams'],
) => reverbUrlHelper.getTrackingPixelSrc(reverbParams);
```